openfisca_core/data_storage/on_disk_storage.py

```diff
@@ -1,85 +1,124 @@
+from __future__ import annotations
+
+from typing import Any, NoReturn, Optional, Sequence
+
 import os
 import shutil
 
 import numpy
 
-from openfisca_core import periods
-from openfisca_core.indexed_enums import EnumArray
+from openfisca_core import periods, indexed_enums as enums, types
+
+from ._enums import Enums
+from ._files import Files
 
 
 class OnDiskStorage:
-    """
-    Low-level class responsible for storing and retrieving calculated vectors on disk
+    """Class responsible for storing and retrieving calculated vectors on disk.
+
+    Attributes:
+        _enums: ?
+        _files: ?
+        is_eternal: ?
+        storage_dir: Path to store calculated vectors.
+        preserve_storage_dir: ?
+
+    Args:
```

Comment:
What is the `args` of a class?

Reply:
The constructor arguments: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html